src/constants/index.js

```diff
@@ -1 +1 @@
-export const ACCESS_TOKEN;
\ No newline at end of file
+export const ACCESS_TOKEN = '0.0063591763509206941606032576692';
```

Comment:
You can POST to `https://mock.don.red/weibo/oauth2/access_token` with body `{ "code": 123456 }` to get a random token.

![image](https://user-images.githubusercontent.com/24371073/99899068-562d3880-2ce1-11eb-80ae-5b685bc0c9e3.png)